Source/items.h

```diff
@@ -395,13 +395,13 @@ struct CornerStoneStruct {
 	ItemStruct item;
 };
 
-extern int itemactive[MAXITEMS];
-extern bool uitemflag;
-extern int itemavail[MAXITEMS];
-extern ItemStruct items[MAXITEMS + 1];
+extern ItemStruct Items[MAXITEMS + 1];
+extern int ActiveItems[MAXITEMS];
+extern int AvailableItems[MAXITEMS];
+extern int AvailableItemCount;

```

Comment:
`ActiveItemCount`, this is how many active items are present. The inverse `MAXITEMS - ActiveItemCount` is the number of available spaces.

Reply:
Thanks, fixed :)